.travis.yml

```diff
@@ -60,6 +60,12 @@ matrix:
         - CMAKE_OPTIONS="-DRModular=ON -DTRAVIS_DISABLE_UNIT_TESTS=ON -DTRAVIS_DISABLE_LIBSHOGUN_TESTS=ON"
         - CC=clang
         - CXX=clang++
+    - compiler: clang
+      services: docker
+      env:
+        - CMAKE_OPTIONS="-DScalaModular=ON -DTRAVIS_DISABLE_UNIT_TESTS=ON -DTRAVIS_DISABLE_LIBSHOGUN_TESTS=ON"
+        - CC=clang
+        - CXX=clang++
```

Comment:
this aint' going to work until you dont add scala runtime to the shogun docker sdk